### PR TITLE
Example scene updates with logo update

### PR DESCRIPTION
**Pull request by cre8ivepark**
## Overview
- Some of the example scenes that use MRTK logo image had to be updated. (scale change)
- Polished description panel in the Slider Example scene

## Screenshots - Before & After

2021-06-22 15_54_13-MixedRealityToolkit-Unity - HandInteractionExamples - Universal Windows Platform
2021-06-22 15_55_04-MixedRealityToolkit-Unity - HandInteractionExamples - Universal Windows Platform

**Slider Example scene's description panel polish:**
2021-06-22 17_36_24-MixedRealityToolkit-Unity - SliderExample - Universal Windows Platform - Unity 2

### CODE REVIEW

**Comment by cre8ivepark** [comment]
@davidkline-ms Could you please merge this? Thanks!

File: Assets/MRTK/Examples/Demos/HandTracking/Scenes/HandInteractionExamples.unity [hunk 1]
```diff
@@ -54,14 +54,15 @@ LightmapSettings:
     m_EnableBakedLightmaps: 1
     m_EnableRealtimeLightmaps: 0
   m_LightmapEditorSettings:
-    serializedVersion: 10
+    serializedVersion: 12
```
Comment (keveleigh) [inline Assets/MRTK/Examples/Demos/HandTracking/Scenes/HandInteractionExamples.unity:57]: It looks like there are a lot of serialization changes in here. Would it be possible for the logo updates to be made in Unity 2018?

Comment (cre8ivepark) [reply Assets/MRTK/Examples/Demos/HandTracking/Scenes/HandInteractionExamples.unity:57]: Oh, I see. I'll use 2018.

**Comment by cre8ivepark** [comment]
@keveleigh  Opened a new PR https://github.com/microsoft/MixedRealityToolkit-Unity/pull/10048

**Pull request closed without merging**